src/schema.js

```diff
@@ -197,6 +197,25 @@ const createGifSchema = Joi.object({
 			.uri()
 			.required(),
 		published: Joi.boolean()
+	},
+	params: {
+		token: Joi.string()
+			.required()
+	}
+})
+
+const updatePasswordSchema = Joi.object({
+	body:{
+		newPassword : Joi.string()
```

Comment:
@itzgeebee let's kindly read around centralizing the schema so as to not repeat ourselves for both the `updatePassword` and `createNewUser`

Reply:
@jocrah I found out about how to do this but it seems I might need to touch the authSchema itself. So, I want to know if I can do it on this issue or it might be better if it's going to be a different refactoring for the schema as a whole

Reply:
okay @itzgeebee  that's cool, not really checked it out myself. we can refactor a bit later. I might create an issue for it